### PR TITLE
Lean agile process slice

### PR DESCRIPTION
**Pull request by joaquimadraz**
### Motivation
[Trello card](https://trello.com/c/r9rPeUxu/96-what-we-do-lean-agile-process)
[Zeplin](https://zpl.io/be13znN)

Add new "Lean and Agile Process" slice to replace existing "How do we do the thing right?" slice.

### Test plan

- Navigate to the what we do page;
- Currently "Lean and Agile Process" slice should appear below "How do we get to the right thing?"
- After https://github.com/redbadger/website-honestly/pull/697 is merged, should appear bellow "Working in cross functional teams" slice.

### Pre-merge checklist

- [ ] Documentation
- [ ] Unit tests
- [ ] Reviews
  - [ ] Code review
  - [ ] Design review
- [ ] Manual testing
  - [ ] Windows 7 IE 11
  - [ ] Windows 10 Edge
  - [ ] Mac OS El Capitan Safari
  - [ ] Mac OS El Capitan Chrome
  - [ ] Mac OS El Capitan Firefox
  - [ ] iOS 9 Safari
  - [ ] Android 6 Chrome
  - [ ] Listen to the site on a screen reader
  - [ ] Navigate the site using the keyboard only
- [ ] Tester approved

### CODE REVIEW

File: site/pages/what-we-do/lean-agile-slice/style.css [hunk 1]
```diff
@@ -0,0 +1,355 @@
...
+  text-align: center;
+}
+
+.word {
+  composes: fontS from "../../../css/typography/_fonts.css";
+  composes: serif from "../../../css/typography/_fonts.css";
+  width: 130px;
+  position: absolute;
+  top: 50%;
+  -webkit-transform: translate(0%, -50%);
```
Comment (dpiatek) [inline site/pages/what-we-do/lean-agile-slice/style.css]: we have autoprefixer https://github.com/redbadger/website-honestly/blob/master/package.json#L11 so prefixes are not needed :)

File: site/pages/what-we-do/lean-agile-slice/style.css [hunk 2]
```diff
@@ -0,0 +1,355 @@
...
+  margin: 0 auto;
+  position: relative;
+}
+
+.stepHeader {
+  composes: fontM2 from "../../../css/typography/_fonts.css";
+  composes: serif from "../../../css/typography/_fonts.css";
+  padding-left: 18px;
+  padding-top: 10px;
+  padding-bottom: 20px;
```
Comment (dpiatek) [inline site/pages/what-we-do/lean-agile-slice/style.css]: probably more easily written as a shorthand? `padding: 10px 0 20px 18px;`

File: site/pages/what-we-do/lean-agile-slice/style.css [hunk 3]
```diff
@@ -0,0 +1,355 @@
...
+  .loopContainer {
+    height: 700px;
+    margin: 0;
+  }
+
+  .word {
+    font-size: 30px;
+    width: 100%;
+    -webkit-transform: translate(-50%, -50%);
+    transform: translate(-50%, -50%);
```
Comment (dpiatek) [inline site/pages/what-we-do/lean-agile-slice/style.css]: Comment as above

File: site/pages/what-we-do/lean-agile-slice/index.js [hunk 4]
```diff
@@ -0,0 +1,79 @@
+import React from 'react';
```
[the remark this answers is not in the document]
Comment (joaquimadraz) [reply site/pages/what-we-do/lean-agile-slice/index.js]: 👍 never used flow btw, looks interesting.

[the remark this answers is not in the document]
Comment (dpiatek) [reply site/pages/what-we-do/lean-agile-slice/index.js]: ah cool, after installing packages you should be able to run it with `make flow`

[the remark this answers is not in the document]
Comment (joaquimadraz) [reply site/pages/what-we-do/lean-agile-slice/index.js]: Got it! All good.

**Comment by chris-brookes** [comment]
[PR#698 - What we do: Lean + Agile process](https://trello.com/c/r9rPeUxu/96-pr698-what-we-do-lean-agile-process)

**Comment by chris-brookes** [comment]
@joaquimadraz - I think we need a bit more space between the the header and the text below. You have the same space between the image and header in tablet and desktop view. With bigger fonts in desktop I think the gap is a bit too small. The design also has a larger gap. 
Please increase the margin above the image to 165px on desktop view. (text 100px so the gap will be 65px).

screen shot 2018-03-12 at 14 25 05

**Pull request merged**